tests/test_sample_binaries.py

```diff
@@ -260,6 +260,17 @@ def test_tailcall_display():
     assert output.count("return fseeko(") == 1
 
 
+def test_member_access_is_in_decompiled_code():
+    """Test that arg1#0->_IO_read_ptr, arg1#0->_IO_write_base and arg1#0->_IO_save_base
+    are displayed as member accesses in the decompiled code."""
+    args = ["python", "decompile.py", "tests/coreutils/binaries/sha224sum", "rpl_fseeko"]
+    output = str(subprocess.run(args, check=True, capture_output=True).stdout)
+
+    assert "->_IO_read_ptr" in output
+    assert "->_IO_save" in output
+    assert "->_IO_write_base" in output
```

Comment:
We are gonna have fun this this test in the future :')

Reply:
Should I use instead regexes in assertions in case there are whitespaces between `->` and member names?
Or what exactly does bother you?

I mean it is coreutils binary and Binja reliably recognizes those structs with those names...
Could you elaborate on your concerns here?

Reply:
another acceptance test means another test I got to add to my pytest ignore list :/. You can not reasonably debug any of them.